templates/files/pimcore/phpstan.neon

```diff
@@ -1,22 +1,22 @@
 includes:
   - vendor/phpstan/phpstan-symfony/extension.neon
 
+  # To create a baseline run the following command and then uncomment the include below (make sure to run the baseline
+  # command with the same level as configured in Grumphp)
+  #      vendor/bin/phpstan analyse --configuration=./phpstan.neon --generate-baseline --level=6
+  #- phpstan-baseline.neon
+
 parameters:
-  excludePaths:
-    # Symfony files
-    - %rootDir%/../../../bin/console
-    - %rootDir%/../../../public/index.php
-    # Full var folder, contains all kind of auto-generated files (e.g. cache, generated Pimcore DataObject classes)
-    - %rootDir%/../../../var/*
-    # Configuration written by Pimcore (which is written either to var/ or to config/pimcore/ depending on settings)
-    - %rootDir%/../../../config/pimcore/*
-    # Unit tests
-    - %rootDir%/../../../tests/fixtures/*
-    # Deployer (which would be nice if it is correct too, but does need its own composer install, therefore excluding)
-    - %rootDir%/../../../build/deployer/*
+  paths:
+    - src
```

Comment:
No rootDir needed here?

Reply:
Apparently it was never needed

According to PHPStan docs:
> Relative paths in the paths key are resolved based on the directory of the config file is in.